rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -29,7 +29,10 @@ class HVAC_SYS:
     SYS_9 = "Sys-9"
     SYS_9B = "Sys-9b"
     SYS_10 = "Sys-10"
+    SYS_11B = "Sys-11b"
```

Comment:
There is no SYS11B, Sub types shall be SYS11_1B...

Reply:
Fixed.